src/components/App.tsx

```diff
@@ -11,17 +11,27 @@ import ThankYouPage from './pages/containers/ThankYouPageContainer';
 import SelectedAreasPage from './pages/containers/SelectedAreasPageContainer';
 import WinterBerthPage from './pages/containers/WinterBerthPageContainer';
 import WinterFormPage from './pages/containers/WinterFormPageContainer';
+import FrontPage from './pages/FrontPage/FrontPage';
 
-export interface Props {
-  locale: 'fi' | 'en' | 'sv';
-}
+import { LocaleOpts } from '../types/intl';
+
+type Props = RouteComponentProps<{ locale: LocaleOpts }>;
 
 const localeParam = ':locale(fi|en|sv)';
 
-const App = ({ locale }: Props) => (
-  <IntlProvider locale={locale} key={locale} messages={messages[locale]}>
+const App = ({
+  match: {
+    params: { locale }
+  }
+}: Props) => (
+  <IntlProvider
+    locale={locale}
+    key={locale}
+    messages={messages[locale]}
+    defaultLocale={LocaleOpts.FI}
+  >
     <Switch>
-      <Redirect exact path={`/${localeParam}/`} to={`/${localeParam}/berths`} />
+      <Route exact path={`/${localeParam}`} component={FrontPage} />
       <Route exact path={`/${localeParam}/berths`} component={BerthPage} />
       <Route exact path={`/${localeParam}/selected_berths`} component={SelectedBerthPage} />
```

Comment:
Ugh... `selected_berths` looks really ugly, it should prolly be `selected-berths`